service/hook/gogs/gogs.go

```diff
@@ -82,17 +82,39 @@ func transformCodePushEvent(codePushEvent CodePushEventModel) hookCommon.Transfo
 
 	if !isLastCommitFound {
 		return hookCommon.TransformResultModel{
-			Error: errors.New("The commit specified by 'after' was not included in the 'commits' array - no match found"),
+			Error:      errors.New("The commit specified by 'after' was not included in the 'commits' array - no match found"),
+			ShouldSkip: true,
+		}
+	}
+
+	if len(lastCommit.CommitHash) == 0 {
+		return hookCommon.TransformResultModel{
+			Error:      fmt.Errorf("Missing commit hash"),
+			ShouldSkip: true,
```

Comment:
No, because it basically means it's an invalid message, maybe from something that "looks like, but isn't, Gogs". Gogs should always send this hash... but if it doesn't, obviously we need to treat that as an error. I removed the SholdSkip tho.